README.md

```diff
@@ -100,7 +110,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is a sql template.  Its components are a name and a list of attributes, including an id which is usually automatically assigned.  A sql schema can be thought of as similar to a class in ruby.
+
```

Comment:
Whenever I think of schema, I think of structure of a database. A schema normally includes, a table name, as well as the column headers, and each column's datatype.  It might also included information about these columns such as validations, or uniqueness, etc. Similar to a class in that it represents a blueprint yes, but would caution the extent of that analogy.